src/main/java/pl/javastart/task/components/Memory.java

```diff
@@ -0,0 +1,34 @@
+package pl.javastart.task.components;
+
+import pl.javastart.task.exception.IllegalClockSpeedException;
+import pl.javastart.task.printer.Printer;
+
+import java.util.InputMismatchException;
+
+public class Memory extends OverclockableComponent {
+    private static final int TEMPERATURE_INCREASE = 10;
+
+    private final int capacity;
+
+    public Memory(String model, String manufacturer, String serialNo,
+                  int clockSpeed, int temperature, int maxTemperature, Printer printer, int capacity) {
+        super(model, manufacturer, serialNo, TEMPERATURE_INCREASE, clockSpeed, temperature, maxTemperature, printer);
+        this.capacity = capacity;
+    }
+
+    @Override
+    public void overclock() throws InputMismatchException, IllegalClockSpeedException {
+        super.overclock();
+    }
```

Comment:
Jeżeli tutaj jedynie wywołujesz metodę z klasy nadrzędnej, to nie ma potrzeby zapisywania takiej metody